src/private.h

```diff
@@ -48,7 +54,7 @@ typedef struct _gpuarray_blas_ops gpuarray_blas_ops;
 struct _gpucontext {
   GPUCONTEXT_HEAD;
   void *ctx_ptr;
-  void *private[7];
+  void *private[8];
```

Comment:
Does void* [7] suffice to hold a `ncclResult_t` variable (it's an enum)? I will check this through the static idd.